docs/vidu-docs/首尾帧生视频.md

```
@@ -0,0 +1,88 @@
+| movement_amplitude | String          | 可选  | 运动幅度<br>默认 `auto`，可选值：`auto`、`small`、`medium`、`large`<br>注：使用q2、q3系列模型时该参数不生效 |
+| audio              | Bool            | 可选  | 是否使用音视频直出能力，默认为true，枚举值为：<br>- false：不需要音视频直出，输出静音视频<br>- true：需要音画同步，输出声音的视频（包括台词和音效）<br>注1：仅q3系列模型支持该参数 |
+| bgm                | bool            | 可选  | 是否为生成的视频添加背景音乐。<br>默认：false，可选值 true 、false<br>- 传 true 时系统将从预设 BGM 库中自动挑选合适的音乐并添加；不传或为 false 则不添加 BGM。<br>- BGM不限制时长，系统根据视频时长自动适配<br>- 该参数在q3系列模型中不生效 |
+| payload            | String          | 可选  | 透传参数<br>不做任何处理，仅数据传输<br>注：最多 1048576个字符                      |
+| off_peak           | Bool            | 可选  | 错峰模式，默认为：false，可选值：<br>- true：错峰生成视频；<br>- false：即时生成视频；<br>注1：错峰模式消耗的积分更低，具体请查看[产品定价](https://platform.vidu.cn/docs/pricing)<br>注2：错峰模式下提交的任务，会在48小时内生成，未能完成的任务会被自动取消，并返还该任务的积分；<br>注3：您也可以[手动取消](https://platform.vidu.cn/docs/cancel-task-api)错峰任务 |
+| watermark          | Bool            | 可选  | 是否添加水印<br>- true：添加水印；<br>- false：不添加水印；<br>注1：目前水印内容为固定，内容由AI生成，默认不加<br>注2：您可以通过watermarked_url参数查询获取带水印的视频内容，详情见[查询任务接口](https://platform.vidu.cn/docs/search-task-api) |
+| wm_position        | Int             | 可选  | 水印位置，表示水印出现在图片的位置，可选项为：<br>`1`：左上角<br>`2`：右上角<br>`3`：右下角<br>`4`：左下角<br>默认为：3 |
+| wm_url             | String          | 可选  | 水印内容，此处为图片URL<br>不传时，使用默认水印：内容由AI生成                          |
+| meta_data          | String          | 可选  | 元数据标识，json格式字符串，透传字段，您可以 自定义格式 或使用 示例格式 ，示例如下：<br>{<br>"Label": "your_label",<br>"ContentProducer": "your_content_producer",<br>"ContentPropagator": "your_content_propagator",<br>"ProduceID": "your_product_id", <br>"PropagateID": "your_propagate_id",<br>"ReservedCode1": "your_reserved_code1", <br>"ReservedCode2": "your_reserved_code2"<br>}<br>该参数为空时，默认使用vidu生成的元数据标识 |
+| callback_url       | String          | 可选  | Callback 协议<br>需要您在创建任务时主动设置 `callback_url`，请求方法为 POST，当视频生成任务有状态变化时，Vidu 将向此地址发送包含任务最新状态的回调请求。回调请求内容结构与查询任务API的返回体一致<br>回调返回的"status"包括以下状态：<br>- `processing` 任务处理中<br>- `success` 任务完成（如发送失败，回调三次）<br>- `failed` 任务失败（如发送失败，回调三次）<br>Vidu采用回调签名算法进行认证，详情见：[回调签名算法](https://platform.vidu.cn/docs/callback-signature) |
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)